client/src/lib/user/state/state.ts

```diff
@@ -9,7 +9,14 @@ type PinnedSession = {
   expires: Date;
 };
 
-type UserState = {pinnedSessions: Array<PinnedSession>};
+type CompletedSession = {
+  id: string;
```

Comment:
@gewfy @swemail as a first step I was just trying to store something (id)

Should we also add `completedAt`? any other field you see being important in this case? 🤔

Reply:
Don't we need to lookup sessions anyway by id? We would have all the data then anyways

Reply:
`completedAt` is definitely needed to create a timeline :P

Reply:
It is but I just thought about needing to look it up anyways, we don't know who the host was an such

Reply:
I think I'll add `completedAt` in case we decide to display/fetch only for e.g "the last 3 or so". 
Then we could order/filter that first and gradually fetch it?

Reply:
done [aa29319](https://github.com/29ki/29k/pull/1609/commits/aa293194a5a5500c3f77622f0c2ebcffbacee477)